kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -114,3 +114,41 @@ namespace          domain   cluster    min  max  goal  current  ready
 weblogic-domain-1  domain1  cluster-1  0    4    2     2        2
 weblogic-domain-1  domain1  cluster-2  0    4    0     0        0
 ```
+
+### Scripts to perform a rolling restart of a WebLogic domain or cluster
+
+The `rollDomain.sh` script can be used to perform a rolling restart of the WebLogic Server Pods in a domain managed by the operator. Similarly, `rollCluster.sh` script can be used to perform a rolling restart of the WebLogic Server Pods belonging to a WebLogic cluster in a domain managed by the operator.
+
+The `rollDomain.sh` script updates the value of `spec.restartVersion` attribute of the domain resource.  The operator will then do a rolling restart of the Server Pods in the WebLogic domain once the value of `spec.restartVersion` is updated. You can provide the new value for `spec.restartVersion` as a parameter to the script or the script will automatically generate a new value to trigger the rolling restart. See the script `usage` information by using the `-h` option.
+
+```shell
```

Comment:
Minor nit: I suspect the 'shell' view slightly confusingly highlights keywords like 'for' and '!' in the 'INFO' lines. Maybe don't use the 'shell' keyword?